### ISSUE TITLE
syncWarnings and syncErrors causing rerendering

### ISSUE THREAD

**Issue by mmahalwy** [erikras/redux-form#3453]
Seems that `syncWarnings` and `syncErrors` are causing rerendering due to returning false on a shallow equality check. Take for example:

```
export default compose(
  reduxForm(),
  otherStuff,
  etc
);
```

**Comment by gustavohenke** [comment]
What version are you using?
This was fixed by #3072, published as of v7.0.0.

**Comment by mmahalwy** [comment]
@gustavohenke ah, makes sense, On 6.7.0
